packages/progress-bar/src/Kaizen/ProgressBar.elm

```diff
@@ -0,0 +1,93 @@
+module Kaizen.ProgressBar exposing (Mood(..), view)
```

Comment:
The elm version is simplified compared to the React version (is missing variants). It just has the variant we needed. It doesn't seem as important to implement them due to Elm being contained.

Reply:
Worth exposing `Configuration` just so that consumers don't have to redefine it if they need to pass a value of this type around?